test/property_constraints_spec.js

```diff
@@ -1,18 +1,19 @@
 var propertyConstraints = require(__dirname + '/../lib/property_constraints');
-var Immutable = require('immutable');
 var chai = require('chai');
-var chaiImmutable = require('chai-immutable');
-chai.use(chaiImmutable);
 var expect = chai.expect;
 
+
+var bindDsl = function(fn) {
```

Comment:
Why is this wrapper function necessary?

doesn't bindDsl =~ propertyConstraints?